### PR TITLE
[feature/SCHOOL-131_login_functionality] Added steps for testing login functionality

### PR DESCRIPTION
**Pull request by HeghineKhachatryan**
Added steps for testing login functionality

### CODE REVIEW

File: src/test/resources/features/common/auth/login.feature [hunk 1]
```diff
@@ -1,7 +1,32 @@
 Feature: Login as different users
```
Comment (mkrtichstepanyan) [inline src/test/resources/features/common/auth/login.feature:4]: Please give more description to this feature

File: src/test/resources/features/common/auth/login.feature [hunk 2]
```diff
@@ -1,7 +1,32 @@
...
 
-  Scenario: Check super admin logs successfully - TC 1.7.2
+  @TC1.7.2 @Regression @Smoke
+  Scenario: Check super admin logs successfully
```
Comment (mkrtichstepanyan) [inline src/test/resources/features/common/auth/login.feature]: logs in

File: src/test/resources/features/common/auth/login.feature [hunk 3]
```diff
@@ -1,7 +1,34 @@
...
+  @TC1.7.2 @Regression @Smoke
+  Scenario: Check super admin logs in successfully
     Given Enter email super@gmail.com in login page
     And Enter password Sa1234567+ in login page
     And Click on 'login' button
     Then The user is on super admin page
+
+  @TC1.7.3 @TC1.7.4 @Regression
+  Scenario Outline: Checking displayed error messages while using incorrect credentials during login
+    When Enter <email> and-or <password> fields
```
Comment (mkrtichstepanyan) [inline src/test/resources/features/common/auth/login.feature]: remove "or" from step

File: src/test/resources/features/common/auth/login.feature [hunk 4]
```diff
@@ -1,7 +1,34 @@
...
+    And Click on 'login' button
+    Then See error message
+
+    Examples:
```
Comment (mkrtichstepanyan) [inline src/test/resources/features/common/auth/login.feature:17]: change "username" word to "email"

Extend examples -> include more login cases

**Review by mkrtichstepanyan — CHANGES_REQUESTED** [review]

File: src/test/resources/features/common/auth/login.feature (hunk 1 above)
Comment (HeghineKhachatryan) [reply src/test/resources/features/common/auth/login.feature:4]: done

File: src/test/resources/features/common/auth/login.feature (hunk 2 above)
Comment (HeghineKhachatryan) [reply src/test/resources/features/common/auth/login.feature]: done

**Review by davbalabekyan — APPROVED** [review]

File: src/test/resources/features/common/auth/login.feature (hunk 3 above)
Comment (HeghineKhachatryan) [reply src/test/resources/features/common/auth/login.feature]: done

File: src/test/resources/features/common/auth/login.feature (hunk 4 above)
Comment (HeghineKhachatryan) [reply src/test/resources/features/common/auth/login.feature:17]: done

**Review by mkrtichstepanyan — APPROVED** [review]

**Pull request merged**